### ISSUE TITLE
Start using CheCluster v2alpha1 in DWCO

### ISSUE THREAD

**Issue by metlos** [eclipse-che/che#19450]
This is a subtask of #19448 and depends on #19449.

Once we have defined `CheCluster` `v2alpha1`, we need to drop the use of the preliminary `CheManager` CR in `devworkspace-che-operator` and start using the `CheCluster` `v2alpha1` instead. This should be fairly simple, because v2alpha1 should contain the same config options as CheManager.

**Comment by metlos** [comment]
This is superseded by #19845. Once we get to the point where we need to define the new CR version, we can revisit this issue or create a new one.
